ui/lesson/TransactionsChallenge/index.tsx

```diff
@@ -280,12 +280,15 @@ const TransactionChallenge: FC<ITransactionProps> = ({
           {allTabsData.map(
             (tabData) =>
               tabData[0] === activeView && (
-                <div key={tabData[0]} className="flex h-full flex-col gap-4 ">
-                  <div className="flex h-full max-h-[calc(100vh-447px-47px)] flex-col overflow-auto px-6 pt-6 font-space-mono md:max-h-[calc(100vh-407px)] ">
-                    <Text className=" font-space-mono text-base leading-[22.22px] tracking-[2%]">
+                <div
+                  key={tabData[0]}
+                  className="flex h-[calc(100vh-70px-56px-65px-48px)] flex-col gap-4 overflow-y-auto md:h-[calc(100vh-70px-56px-65px)] short:h-[calc(100vh-70px-56px-65px)]"
```

Comment:
Christoph mentioned magic numbers in the ticket. Are these ones right here going to be an issue?